src/reference/config/writing-platform-configs.rst

```diff
@@ -167,7 +172,7 @@ Background Jobs on Cluster with Other Options
 Submit PBS Jobs from Localhost
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- **Platforms can share hosts and not share batch systems.**
+- **Platforms can share hosts without sharing batch systems.**
```

Comment:
(Off topic, but as-is this could be interpreted as saying "platforms cannot share batch systems"?)